pandas/_libs/tslibs/nattype.pyx

```diff
@@ -180,6 +181,9 @@ cdef class _NaT(datetime):
         # Includes Period, DateOffset going through here
         return NotImplemented
 
+    def __radd__(self, other):
```

Comment:
can you conditionally add these if we are using cython >= 3?

Reply:
You don't have to. Just leave them in. Doesn't hurt much.